argocd-mixin/makefile

```
@@ -0,0 +1,27 @@
+JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 1 --string-style s --comment-style s
+
+.PHONY: all
+all: dashboards_out prometheus_alerts.yaml
+
+.PHONY: fmt
+fmt:
+		find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | xargs -n 1 -- $(JSONNET_FMT) -i
+
+.PHONY: lint
+lint:
+		find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
+			while read f; do \
+					$(JSONNET_FMT) "$$f" | diff -u "$$f" -; \
+            done
+		mixtool lint mixin.libsonnet
+
+dashboards_out: mixin.libsonnet config.libsonnet $(wildcard dashboards/*)
```

My mistake, I left some superfluous config in this file. Thank you for adding it!

```suggestion
dashboards_out: mixin.libsonnet $(wildcard dashboards/*)
```